modules/doc/content/framework_development/git.md

```diff
@@ -20,13 +20,15 @@ however, you might find yourself with a huge number of remotes that you don't re
 
 ## Git completion
 
-To enable bash completion for git (so you can tab-complete 'git co' or 'git br' commands) download [git_completion.sh](http://mooseframework.org/static/media/uploads/files/git_completion.sh) to your home directory, and append the following line to the end of your ~/.bash_profile (or ~/.bashrc)
+#### Using the bash shell
+
+To enable bash completion for git (so you can tab-complete 'git co' or 'git br' commands) download [git_completion.sh](https://github.com/git/git/blob/master/contrib/completion/git-completion.bash) to your home directory, and append the following line to the end of your ~/.bash_profile (or ~/.bashrc)
 
 ```bash
 source ~/git_completion.sh
 ```
 
-If you would like to have the name of the current branch in your prompt, download [git_prompt.sh](http://mooseframework.org/static/media/uploads/files/git_prompt.sh) to your home directory, and append the following lines to the end of your ~/.bash_profile (or ~/.bashrc)
+If you would like to have the name of the current branch in your prompt, download [git_prompt.sh](https://github.com/git/git/blob/master/contrib/completion/git-prompt.sh) to your home directory, and append the following lines to the end of your ~/.bash_profile (or ~/.bashrc)
 
 ```bash
 source ~/git_prompt.sh
```

Comment:
```suggestion
source ~/git-prompt.sh
```

Forgot this one

Reply:
are you sure? it looks like the underscore was changed to a hypen in the github repo address?

Reply:
My proposed change modifies the underscore and turns it to a hyphen, though - just like the github repo address.

Reply:
oh, I missed the second o and read "forget this one"